src/pgroonga.c

```diff
@@ -5348,13 +5349,14 @@ PGrnSearchBuildConditionPrepareConditionBuildMatchColumns(
 		}
 		if (weight != 1)
 		{
-			grn_expr_append_const_int(
-				ctx, matchColumns, weight, GRN_OP_PUSH, 1);
-			PGrnCheck("%s failed to push weight: <%s>[%d] * <%d>",
-					  tag,
-					  indexName,
-					  section,
-					  weight);
+			PGrnExprAppendConstInteger(matchColumns,
+									   weight,
+									   GRN_OP_PUSH,
+									   1,
+									   tag,
+									   "index: <%s> section: <%d>",
```

Comment:
`index[section] * weight`でマルチカラムインデックスの`section`番目のソースを重み`weight`で検索するという`match_columns`のシンタックスになるのでこうでいいっす。

```suggestion
									   "index: <%s>[%d]",
```